### PR TITLE
Added instructions for offline discovery import

### PR DESCRIPTION
**Pull request by dennis-brinley**
### What is the purpose of this change?

Provide more detailed instructions for importing scan output from off-line agents into Event Portal. Including instructions for modifying IDs associated with the original scan to the target agent/service.

### How was this change implemented?

Updated docs/rest.md after "manual import"

### How was this change tested?

N/A - Documentation Only

### Is there anything the reviewers should focus on/be aware of?

    ...

### CODE REVIEW

File: docs/rest.md [hunk 1]
```diff
@@ -593,4 +593,59 @@ A new file with the following naming convention
...
+## Manual Import of Off-Line Agent Discovery
+The results of a scan run against one messaging service may be uploaded for audit and import to another messaging service defined in Event Portal. 
+
+### Procedure
+In essence, you will perform a scan from one agent, export it, and upload it to Event Portal using another agent. However, the exported data contains identifiers specific to the agent that performed the scan. These identifiers must be modified to suit the agent performing the upload, and modified to the correct messaging service.
+
+### Requirements
+- Source Configuration (off-line):
+    - EMA configuration (e.g. `your-source-config.yml`)
+    - Stand-alone setting path: `eventPortal.gateway.messaging.standAlone: true`
```
Comment (moodiRealist) [inline docs/rest.md:607]: please change `standAlone` to `standalone` to match the config name

File: docs/rest.md [hunk 2]
```diff
@@ -593,4 +593,59 @@ A new file with the following naming convention
...
+    - EMA configuration (e.g. `your-target-config.yml`)
+- Retain the following settings from the EMA for **target** configuration:
+    - **messagingServiceId** - Path in EMA config: `plugins.resources.id`
+    - **emaId** - Path in EMA config: `eventPortal.runtimeAgentId`
+
+### Detailed Procedure
+1. Execute source message service scan
+    1. Initiate the source scan from EMA REST API; e.g.:<br>
+        ```bash
+        curl -H "Content-Type: application/json" -X POST http://localhost:8180/api/v2/ema/messagingServices/{msgServiceId}/scan -d '{"scanTypes": ["KAKFA_ALL"], "destinations":["FILE_WRITER"]}'
```
[the remark this answers is not in the document]
Comment (dennis-brinley) [reply docs/rest.md]: done

File: docs/rest.md (hunk 1 above)
Comment (dennis-brinley) [reply docs/rest.md]: done

**Review by moodiRealist — APPROVED** [review]

File: docs/rest.md [hunk 3]
```diff
@@ -593,4 +593,59 @@ A new file with the following naming convention
...
+        ```bash
+        ## Update the META_INF.json in the zip file
+        zip file_name.zip META_INF.json
+        ```
+4. Upload the scan file to the target
+    1. If you are using two separate EMA configuration files, one for the source scan and one for the target upload (which you probably are), you will likely need to make sure that configuration file for the target EMA instance is active. Such as stop the Event Management Agent and re-run specifying the alternate target.
+    2. Execute the upload<br>
+        ```bash
+        curl --location --request POST 'http://localhost:8180/api/v2/ema/resources/import' \
+            --form 'file=@file_name.zip'```
```
Comment (moodiRealist) [inline docs/rest.md:650]: please double check this part. Seems like the rendered md file will have extra ` ``` ` :
Screen Shot 2023-04-27 at 10 19 13 PM

Comment (dennis-brinley) [reply docs/rest.md:650]: Corrected by adding newline to render as block code

**Pull request merged**